src/meta/client/MetaClient.cpp

```diff
@@ -39,8 +39,25 @@ MetaClient::~MetaClient() {
 }
 
 
-void MetaClient::init() {
+void MetaClient::waitForMetadReady() {
     loadDataThreadFunc();
```

Comment:
Let's figure out what's ready for Meta client. 
Currently,  meta client sends HB to meta server on storage layer,  but it does not on query engine side. 
On storage side,  meta client ready means that server accept the HB and local cache is filled. 
On graph side,  meta client ready means that local cache is filled. 

So we should handle both situations above.

Reply:
Modified! Please review!